### DISCUSSION TITLE
No way to select date range in layout v2

### DISCUSSION THREAD

**Discussion by Syphdias** [firefly-iii/firefly-iii#5370] [Firefly III] [answered]
There seems to be no way to select a date range in layout v2, e.g. in the account transactions.

In Layout v1
![image](https://user-images.githubusercontent.com/16988672/144909148-9471aa00-8b45-4e38-b572-a749249c4492.png)

I also noticed #4618 was closed a while ago with open issues and no mention in the most recent releases and the roadmap https://github.com/firefly-iii/firefly-iii/projects/6 is empty. Is this still in development, was it dropped, or is it done?

**Comment by JC5** [comment, marked as answer]
Click the dragon in the top right corner. A new public roadmap is coming.

**Comment by Syphdias** [comment]
Ah, thanks! But apparently not on all sites yet:
![image](https://user-images.githubusercontent.com/16988672/144926440-26529eb5-6fb0-4bc6-a44c-e53ee6c017d6.png)
Version: 5.6.5
